Todolist/index.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+import "./homework_1.css";
+import {add_button} from "./button_add.js";
+import {hide_button, show_button}  from "./buttons.js";
+import {createListItem} from "./list.js"
+
+axios.get('https://learn-front-end-api-212606.appspot.com/api/v1/todos')
+  .then(function (response) {
+    if(response.data.success){
+        response.data.todos.forEach(element => {
+            createListItem(element);
+        });
+    }
+    console.log(response);
+  })
+  .catch(function (error) {
+    console.log(error);
+  });
+
+window.addEventListener('load', () => {
+    add_button();
```

Comment:
Name of this 3 methods sound like call to action. If I see this line of code, I think that button should be created.
Better to put your event listeners to some method with name `initEventHandlers` and call in on window load